### PR TITLE
Issue #7: Added Prettier

### LINKED ISSUE #7: GitHub: Add prettier action in github action
[Prettier](https://prettier.io/) is a code formatting tool that would basically change codes that are easier to understand and looks good.
While there are multiple ways this can be (plugins, running prettier CLI on file etc), this plugin would change the code of a PR if prettier is used before on code.

The action can be found here:- https://github.com/marketplace/actions/prettier-action
Steps to do:-
1. Create a `prettier.yml` in [.github/workflows](https://github.com/LNMIIT-CODING/stonks-frontend/tree/master/.github/workflows)
2. Add code
3. Squash commit changes. Commit message must be `Issue #XXXX: DESCRIPTION OF CHANGE`
4. Make a PR

Points to remember -
1. If you are an admin, please fork and make changes in your fork.
2. Since, this is an Issue with GitHub tag, GitHub rules will be strictly followed (Only single commit message, commit message format etc).

Target of this Issue - 
1. Understand basic of GitHub, Github action, unix.
2. GitHub rules and guidelines for a good developer.

Please mention I am on it. One of the admin would assign to you.

### PR DESCRIPTION
**Pull request by OmGuptaIND**
Added Prettier Action this time only a file named prettier.yml is here

Please review and merge

### CODE REVIEW

**Review by AkMo3 — APPROVED** [review]

**Comment by AkMo3** [comment]
Approved, I will squash merge after mayank approves.

**Review by DreadArceus — APPROVED** [review]

**Pull request merged**
